### ISSUE TITLE
`set_property` validation

### ISSUE THREAD

**Issue by Xterionix** [Blockception/BC-Minecraft-Bedrock-Diagnoser#186]
- Check if the property exists on the entity
- Check if the property supports the value being set

**Comment by DaanV2** [comment]
Just to clarify, this is about the set_property in entity events right?

**Comment by Xterionix** [comment]
Yes, could extend to has_property in commands if you haven't already.

**Comment by DaanV2** [comment]
I look, but it should already be doing this 🤔 
Unless you got me a scenario that mismatches

**Comment by Xterionix** [comment]
I have not yet used has_property so I wouldn't know but if I believe there's something missing from it, I'll open an issue.
